data/json/recipes/practice/skills.json

```
@@ -0,0 +1,47 @@
+[
+  {
+    "id": "prac_computer_beg",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (beginner)",
+    "description": "Practice the basics of using a computer",
+    "skill_used": "computer",
+    "time": "30 m",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "autolearn": [ [ "computer", 1 ] ],
+    "book_learn": [ [ "mag_computer", 0 ], [ "howto_computer", 0 ] ],
+    "tools": [ [ [ "laptop", 25 ], [ "eink_tablet_pc", 10 ], [ "smart_phone", 10 ] ] ]
+  },
+  {
+    "id": "prac_computer_int",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (intermediate)",
+    "description": "Practice using the command line, writing and running scripts making use of common algorithms and data structures",
+    "skill_used": "computer",
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 4, "skill_limit": 5 },
+    "autolearn": [ [ "computer", 4 ] ],
+    "book_learn": [ [ "manual_computers", 2 ], [ "computer_science", 3 ], [ "textbook_computer", 3 ] ],
+    "tools": [ [ [ "laptop", 50 ] ] ]
+  },
+  {
+    "id": "prac_computer_adv",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (advanced)",
+    "description": "Practice coding advanced algorithms and software exploits",
+    "skill_used": "computer",
+    "time": "90 m",
```

```suggestion
    "time": "1 h",
```
We should probably keep these to a standard 1 hour and only deviate when there's an obvious reason it should be different.